backend/src/test/groovy/com/capstone/moneytree/model/SanitizedStockTest.groovy

```diff
@@ -0,0 +1,44 @@
+package com.capstone.moneytree.model
+
+import com.capstone.moneytree.model.node.User
+import com.capstone.moneytree.model.node.Stock
+import com.capstone.moneytree.model.relationship.Owns
+import org.junit.Test
+
+import static com.capstone.moneytree.utils.MoneyTreeTestUtils.*
+
+import spock.lang.Specification
+
+import java.sql.Date;
+import java.time.Instant;
+
+/**
+ * Unit Tests for the SanitizedStock model.
+ */
+class SanitizedStockTest extends Specification {
+    @Test
+    def "it should create a SanitizedStock from an Owns releationship"()
+
+    {
```

Comment:
Move this bracket on the same line as the function name.

Reply:
I think this is the last comment I have before we merge. This is a high priority tasks so let`s try to merge it today 🥇

Reply:
I'll fix the indentation for the last comment tonight. There's another commit to this pr which is a bit of more work on frontend, it is almost done, I just need to wrap things up and push, I'll do tonight and then merge. Please approve :-)